test/CMakeLists.txt

```diff
@@ -986,6 +992,9 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --ver
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 1 8 8 --weights 1 1 2 2 --pads_strides_dilations 0 0 1 1 2 2 --disable-forward --disable-backward-data
 COMMAND ${DYNAMIC_IMPLICITGEMM_WRW_ENVS_XDLOPS} $<TARGET_FILE:test_conv2d> --verbose --half --input  1 128 56 56 --weights 1 128 5 5 --pads_strides_dilations 0 0 2 2 1 1 --disable-forward --disable-backward-data
 )
+add_custom_test(test_conv_igemm_ck_xdlops_bwd_v4r1 SKIP_UNLESS_ALL HALF_ENABLED GFX908_ENABLED VEGA_DISABLED
```

Comment:
If we do not need this for FP32, then let's add FLOAT_DISABLED.

Reply:
We already have `test_regression_half_vega` -- a place where FP16-only regression tests for Vega10/20 can be added:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/fca9fed0b571b1b8ff587f25a75221d36e62e8b0/test/CMakeLists.txt#L1082-L1089

Maybe it's worth to proceed the same way? I mean:
- Rename this test to `test_regression_half_mi100`
- Move environment settings inline (just after COMMAND) and remove CK_IMPLICITGEMM_BWD_V4R1_ENVS_XDLOPS
- Add a comment like `# Regression test for SWDEV-291202`